### DISCUSSION TITLE
Lightbox on slide click

### DISCUSSION THREAD

**Discussion by vitalijalbu** [nolimits4web/swiper#4336] [Ideas]
Would be fine to create a lightbox onclik of a slide :D; just to make this library more beautiful and complete of everything

**Comment by Psytanium** [comment]
Interested in this feature too.

**Comment by stephangriesel** [comment]
Agree, would be nice to have this option

**Comment by Arhell** [comment]
https://codepen.io/ezra_siton/pen/XNpJaX

**Reply by vitalijalbu** [reply]
yes, but only with swiper js. no other external libs :D

**Reply by inttyl** [reply]
I am looking for such functionnality too. 
For now I understand that when we click on an image there are no options to view the album as a lightbox

**Reply by pmorch** [reply]
As I wrote in #5698 where this codepen was mentioned:

As far as i can see, https://codepen.io/ezra_siton/pen/XNpJaX does something different:

It uses [swiper on the "main page"](https://github.com/user-attachments/assets/be796934-379c-4c5a-9657-45b9cdd3daee), and then when you click on an image, it opens [Photoswipe in a lightbox](https://github.com/user-attachments/assets/507ae24c-a485-4192-b273-404a9c074b44).

What the OP is asking for is how to open **Swiper** in a overlay. https://codepen.io/ezra_siton/pen/XNpJaX opens **Photoswipe** in an overlay. Not the same thing at all.

**Comment by webdiva69** [comment]
Would also love this feature

**Comment by pmorch** [comment]
As I wrote in #5698:

I just created https://codesandbox.io/p/sandbox/2hgrlw that does this as a proof of concept.

It is functional and shows how one can use the native `<dialog>` HTML element to achieve this, but a lot of styling is still required to get this to look great. Compare my  sandbox screenshot:

![my sandbox screenshot](https://github.com/user-attachments/assets/08210d7d-e18a-408b-bf14-170f479d961c)

to Photoswipe in a lightbox:

![Photoswipe in a lightbox](https://github.com/user-attachments/assets/507ae24c-a485-4192-b273-404a9c074b44).

So this is a proof-of-concept - nothing more.
